modules/cli-installing-cli-rpm.adoc

```diff
@@ -43,6 +43,15 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
 ----
 
 . Enable the repositories required by {product-title} {product-version}.
+
+** For Red Hat Enterprise Linux 9:
++
+[source,terminal]
+----
+# subscription-manager repos --enable="rhocp-4.11-for-rhel-9-x86_64-rpms"
```

Comment:
@zhouying7780 If we are going to call oc supported with RHEL 9 for 4.11 (see question here: https://issues.redhat.com/browse/OSDOCS-3800?focusedCommentId=20558750&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20558750) can you test and see what the correct repo is for installing via RPM for oc on RHEL 9?

Reply:
after install the bash-completion package on RHEL9 , I could install the oc client by yum repo:
cat /etc/yum.repos.d/openshift.repo
[ose-v4-client]
name=ose-v4-client
baseurl=http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/plashets/4.11/building/x86_64/os/
enabled=1
gpgcheck=0
metadata_expire=1